src/raphael_backend_flask/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class FlashException(Exception):
```

Comment:
What is a `FlashException` and why can't it be handled with a `ValueError` or something equally appropriate?

Reply:
If you look at `handle_exception` in routes.py, it should make sense. It’s an expected exception, that includes a category that’s used for display purposes.

Reply:
Indeed - it's used as a way of indicating that exception should be shown to the user using the "flash" that appears at the top of the page. It's not great from an error semantics point of view, but it works well enough for use case here.